portfolio/src/main/webapp/style.css

```diff
@@ -174,17 +174,15 @@ a:hover{
   height: 0;
   opacity: 0;
 }
-/* 
-.text:hover{
-  color: white;
-  font-size: 20px;
-  position: absolute;
-  top: 50%;
-  left: 50%;
-  -webkit-transform: translate(-50%, -50%);
-  -ms-transform: translate(-50%, -50%);
-  transform: translate(-50%, -50%);
-  text-align: center;
-  opacity: 1;
-} */
+
+
```

Comment:
Let's be consistent. There is no reason to keep two spaces between styles. Let's keep single empty line between them.

Reply:
I'm not sure if i did this correct, because I was working on the branch already, but I fixed the doubled spaces and also push new changes